### PR TITLE
Workshop ID#142 new Lab 13: "Public and Private Network Access in Autonomous Database"

### PR DESCRIPTION
**Pull request by greenrick**
Workshop ID#142 "Manage and Monitor Autonomous Database", new Lab 13: "Public and Private Network Access in Autonomous Database". 
This lab focuses on showing how to limit network access using private endpoints.

### CODE REVIEW

File: shared/workshops/freetier-indepth-test-private-endpoints/intro.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Introduction
+
```
Comment (anooshapilli) [inline shared/workshops/freetier-indepth-test-private-endpoints/intro.md:2]: @greenrick - Add heading 2 - "About this Workshop" after the Introduction

File: shared/workshops/freetier-indepth-test-private-endpoints/intro.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+## What is an Autonomous Database?
+Oracle Autonomous Data Warehouse and Oracle Autonomous Transaction Processing are built around the market leading Oracle database and come with fully automated data warehouse and transaction processing specific features that deliver outstanding query performance. This environment is delivered as a fully managed cloud service running on optimized high-end Oracle hardware systems.  You don’t need to spend time thinking about how you should store your data, when or how to back it up or how to tune your queries. We take care of everything for you.
+
+Watch our short video that explains key features in Oracle's Autonomous Database:
+
+[](youtube:c-DUIePFKco)
+
+Oracle’s Autonomous Database is the perfect quick-start service for fast data loading and sophisticated data reporting and analysis. Oracle manages everything for you so you can focus on your data.
+
+## Workshop Objectives
```
Comment (anooshapilli) [inline shared/workshops/freetier-indepth-test-private-endpoints/intro.md:18]: Remove the word Workshop before the Objectives and Prerequisites

File: shared/workshops/freetier-indepth-test-private-endpoints/intro.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+*Note: If you have a **Free Trial** account, when your Free Trial expires your account will be converted to an **Always Free** account. You will not be able to conduct Free Tier workshops unless the Always Free environment is available. **[Click here for the Free Tier FAQ page.](https://www.oracle.com/cloud/free/faq.html)***
+
+**You are all set; let's begin! Click Getting Started in the Contents menu on the left.**
+
+## Learn More
+
+Use these links to get more information about Oracle Autonomous Database:
+
+- <a href="https://www.oracle.com/database/autonomous-database.html" target="\_blank">Oracle Autonomous Database website</a>
```
Comment (anooshapilli) [inline shared/workshops/freetier-indepth-test-private-endpoints/intro.md:41]: @greenrick - Please use the markdown format to reference the links

**Review by anooshapilli — COMMENTED** [review]
@greenrick - Please make the requested in-line changes to the workshop

**Comment by greenrick** [comment]
@anooshapilli - I fixed the 3 issues you raised. Actually, you found them in a temporary testing folder that I should have deleted, and which I just now deleted: shared/workshops/freetier-indepth-test-private-endpoints/. However, I made the changes that you raised in the actual folders for production.

Workshops 141 and 142 fixed items pointed out by Anoosha Pilli from 1st pull request, mostly in intro.md files
- Added "About this Workshops" under "Introduction"
- Removed "Workshop" from "Workshop Objectives" and "Workshop Prerequisites"
- removed 2 temporary folders in which Anoosha found things needed fixing, but made those fixes in the actual production folders. Removed files from "\adb\workshops\freetier-indepth-test-private-endpoints" and entirely removed "freetier-indepth-test-create-clone-from-backup" (at the moment, there's a file open somewhere on my local PC that doesn't allow me to delete the empty \adb\workshops\freetier-indepth-test-private-endpoints folder.)

**Review by anooshapilli — APPROVED** [review]

**Pull request merged**